### PR TITLE
ENT-1085 Revert changes related to ENT-1052

### PR DESCRIPTION
**Pull request by brittneyexline**
We are currently seeing no enterprise offers being applied in the basket page in production. To be sure that we will use the previous offers logic, I am reverting the feature changes in ecommerce. The lms will still send the catalog param but the intent of this revert is that ecommerce will do nothing with the param.

### CODE REVIEW

File: ecommerce/extensions/basket/utils.py [hunk 1]
```diff
@@ -271,31 +269,6 @@ def basket_add_organization_attribute(basket, request_data):
 
 
 @newrelic.agent.function_trace()
```
Comment (robrap) [inline ecommerce/extensions/basket/utils.py]: Did you mean to move this trace and not remove it

Comment (brittneyexline) [reply ecommerce/extensions/basket/utils.py]: ah, looks like i should have removed it. thanks!

**Review by bill-filler — APPROVED** [review]
approved

**Comment by brittneyexline** [comment]
@bill-filler @robrap is there a reason quality checks would start failing for ecommerce/static/js/payment_processors/cybersource.js (especially since that file didn't change)?
that appears to be why the checks are failing...

**Pull request merged**